core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/DrmSessionErrorForwarder.java

```diff
@@ -0,0 +1,38 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.google.android.exoplayer2.drm.DefaultDrmSessionManager;
+import com.novoda.noplayer.Player;
+import com.novoda.noplayer.exoplayer.playererror.DrmInitiatingError;
+import com.novoda.noplayer.listeners.ErrorListeners;
+
+class DrmSessionErrorForwarder implements DefaultDrmSessionManager.EventListener {
+
+    private final Player player;
+    private final ErrorListeners errorListeners;
+
+    DrmSessionErrorForwarder(Player player, ErrorListeners errorListeners) {
+        this.player = player;
+        this.errorListeners = errorListeners;
+    }
+
+    @Override
+    public void onDrmKeysLoaded() {
+        // TODO: Are we interested?
```

Comment:
@ouchadam what would be nice use case of this not-used callbacks? 

If there are none I would suggest replacing the TODO with a `not-used` comment